### DISCUSSION TITLE
Aspire Authorization: Microsoft Entra External ID

### DISCUSSION THREAD

**Discussion by aitrailblazer** [microsoft/aspire#5582] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Losing the socket connection.

I followed:

https://learn.microsoft.com/en-us/training/modules/create-aspire-applications/learn-how-create-new-aspire-application

How to add Authorization with External ID?

Overview - External ID in external tenants - Microsoft Entra External ID | Microsoft Learnhttps://learn.microsoft.com/en-us/entra/external-id/customers/overview-customers-ciam
Secure your apps using External ID in an external tenant

Issue description
When Microsoft Entra External ID Authorization is added the demo app loses interactivity (Counter) is not working
and there is a browser message the socket loses connection.

Expected behavior [What you expected to happen.]

Adding Microsoft supported Authorization should be working.

Actual behavior [What actually happened.]

Doesn't works at all
![image](https://github.com/user-attachments/assets/45725975-e9d2-4f18-bd41-dc9dfdd8a53d)

### Expected Behavior

The app should work as expected.

### Anything else?

https://github.com/microsoft/azure-container-apps/issues/1236

**Comment by davidfowl** [comment]
It's unclear if this is aspire related. If you run the application outside of the aspire host does it work?
